libpromises/generic_agent.h

```diff
@@ -87,6 +87,7 @@ typedef struct
             char *bootstrap_port;
             char *bootstrap_ip;
             bool bootstrap_trust_server;
+            bool bootstrap_trigger_policy;
```

Comment:
maybe invert this as well, so it's all skip, like the option? I dunno.

Reply:
I was thinking the same, but then I saw we had a "constructor" that sets the defaults. So there's no technical reason to have it inverted and I think it is better readable like this.